scripts/deploy_taco_application.py

```diff
@@ -14,15 +14,20 @@ def main(account_id=None):
     deployments_config = DEPLOYMENTS_CONFIG
 
     if CURRENT_NETWORK in LOCAL_BLOCKCHAIN_ENVIRONMENTS:
-        _, _, t_staking, _, _ = deploy_mocks(deployer)
+        _, _, t_staking, _, _, t_token = deploy_mocks(deployer)
     else:
         t_staking = deployments_config.get("t_staking")
+        t_token = deployments_config.get("t_token")
 
-    simple_pre = project.SimplePREApplication.deploy(
+    # TODO deploy proxy
```

Comment:
Opened an issue to track this: https://github.com/nucypher/nucypher-contracts/issues/105